product-scenarios/3-securing-an-api-with-different-security-protocols/3.2-api-developers-want-to-secure-their-api-using-an-user-access-control-machanism/3.2.1-api-developer-wants-to-secure-api-resources-using-user-roles/src/test/java/org/wso2/am/scenario/tests/api/secure/userRoles/SecureUsingUserRolesTestCases.java

```diff
@@ -136,13 +182,42 @@ private void deleteRoles() throws APIManagementException {
         }
     }
 
+    private void createAndPublishAPI()throws APIManagerIntegrationTestException{
```

Comment:
Shall we use this common method for this.
https://github.com/wso2/product-apim/blob/cd655777f335d12a56f15e377637c8ebf0f08e7a/product-scenarios/scenarios-common/src/main/java/org/wso2/am/scenario/test/common/APIPublisherRestClient.java#L928

Reply:
Done